lib/tests/streamlit/delta_generator_test.py

```diff
@@ -139,6 +139,7 @@ def test_public_api(self):
                 "page_link",
                 "plotly_chart",
                 "progress",
+                "popover",
```

Comment:
nit: `popover` should be sorted before `progress`

Reply:
Moved it up. But I think this doesn't really matter here since it is anyways added to a set.

Reply:
That might be true from a technical perspective, but I think from a code hygiene perspective, having a sorted list might help a developer to read / scan / understand the list.